server/routes/dashboard.js

```diff
@@ -13,38 +13,33 @@
  * permissions and limitations under the License.
  */
 
-import { Server, Request, ResponseToolkit } from 'hapi';
+var XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest;
 
-const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
-
-  export default function (server) {
-    const { callWithRequest } = server.plugins.elasticsearch.getCluster('data'); 
-
-    server.route({
-      path: '/api/reporting/get_dashboards',
-      method: 'GET',
-      async handler(req, h) {
-        try {
-          var dashboards = getDashboards();
-        }
-        catch (e) {
-          console.log(e);
-        }
-        return dashboards;
+export default function(server) {
+  server.route({
+    path: '/api/reporting/get_dashboards',
+    method: 'GET',
+    async handler() {
+      try {
+        var dashboards = getDashboards();
+      } catch (e) {
+        console.log(e);
       }
-    });
-  }
+      return dashboards;
+    },
+  });
+}
 
-  function getDashboards() {
-    var jsonBeforeParse = getJson();
-    var jsonAfterParse = JSON.parse(jsonBeforeParse);
-    return jsonAfterParse;
-  }
+function getDashboards() {
+  var jsonBeforeParse = getJson();
+  var jsonAfterParse = JSON.parse(jsonBeforeParse);
+  return jsonAfterParse;
+}
 
-  function getJson() {
-      var url = "http://localhost:9200/.kibana/_search?q=type:dashboard";
-      var HttpRequest = new XMLHttpRequest();
-      HttpRequest.open("GET", url, false);
-      HttpRequest.send(null);
-      return HttpRequest.responseText; 
-  }
\ No newline at end of file
+function getJson() {
+  var url = 'http://localhost:9200/.kibana/_search?q=type:dashboard';
```

Comment:
I think we shouldn't hardcode `localhost:9200` for Elasticsearch- use `server.plugins.elasticsearch.getCluster('data');` instead

Reply:
That's part is not implemented yet. I just leave the code as it is. I think I'll just delete those files for now, will plan those parts for future PRs

Reply:
Sounds good 👍